packages/stencil/src/components/datacom-radio/datacom-radio.css

```diff
@@ -67,7 +67,7 @@
 .dc-radio-radio .dc-radio-input:focus:focus-visible {
   outline: 3px solid var(--dc-primary-colour);
   border-radius: 12px;
-  outline-offset: 1px;
+  outline-offset: px;
```

Comment:
You don't have a value set here. Pretty sure this isn't valid CSS. Do you still need this outline-offset then?

Reply:
Thanks for spotting. I have added.